### DISCUSSION TITLE
Deploy ortoni report in Github

### DISCUSSION THREAD

**Discussion by rahulbendale03** [ortoniKC/ortoni-report#15] [Ideas]
Hi @ortoniKC 
Can we deploy this report in github using github actions as a Artifacts?

**Comment by ortoniKC** [comment]
It is in the pipeline for future release.

**Reply by rahulbendale03** [reply]
Thanks

**Comment by ortoniKC** [comment]
It should work now, pls check.

**Comment by rahulbendale03** [comment]
Have you creates reference doc for this

**Reply by ortoniKC** [reply]
No, it should be the default of the playwright report. Instead of the playwright report give the Ortoni report folder path in the config.

**Comment by rahulbendale03** [comment]
Ok, thanks for the clarification.
